src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -688,13 +704,19 @@ export default class ElementWrapper extends Wrapper {
 		`);
 
 		binding_group.events.forEach(name => {
-			if (name === 'elementresize') {
-				// special case
+			if (name.startsWith('elementresize')) {
```

Comment:
Just to be sure - can this be called with anything else than the four options below? If yes, should there be a `if (!functionName) { return } ` or `if(!functionName) { throw new Error('can only ...') }`?

Reply:
Not sure, but fixed anyway.